Source/Components/Touchpoints/Types/FlexCoverCarousel/Card/MLBusinessFlexCoverCarouselItemView.swift

```diff
@@ -105,12 +120,17 @@ public class MLBusinessFlexCoverCarouselItemView: UIView {
         guard let mainTitleTopText = text else { return }
         mainTitleTopLabel.text = mainTitleTopText
         mainTitleTopLabel.textColor = getLabelUIColor(color)
+        mainTitleTopLabel.setLineHeight(18)
+        headerContainer.addArrangedSubview(mainTitleTopLabel)
     }
     
     private func createMainSubtitle(with text: String?, color: String?) {
-        guard let mainSubtitleText = text else { return }
+        guard let mainSubtitleText = text,
+              mainSubtitleText != "",
+              mainSubtitleText != " " else { return }
```

Comment:
Están buenas las validaciones, pero son escenarios válidos que nos lleguen estos strings así desde BE?

Reply:
de paso, el primero podes hacer `mainSubtitleText.isEmpty`

Reply:
Sumo una cosa ás jejo, estoy de acuerdo con los chicos, peeero en caso de que por alguna razón querramos contemplarlo sería mejor haciendo un trim y después si es empty. Onda `mainSubtitleText.trimmingCharacters(in: .whitespacesAndNewlines).isEmpty`.

Reply:
Jaja, lo de "" es una validación que agregamos porque ya hubo problemas con eso. La otra validación la agregué de miedoso, ya la saco y me gusta lo que pasó @tomidelucca

Reply:
@dylanmuszelml  Muy buena!, no sabía de la existencia

Reply:
Fla, la condición `mainSubtitleText != " "` no hay que ponerla, no es parte del caso de uso. La otra condición está ok.

Reply:
Para darles contexto a todos, el caso real es que solo debería ocultarse cuando sea `nil`, pero en su momento tuvimos un problema de reciclado que se mostraba el subtitle de otras cards cuando venía no venia en otra, lo resolvimos desde backend enviando el mainSubtitleText = "" para dejar el espacio en blanco. Ahora queremos que se oculte cuando no venga, así que es el caso de `nil` y de `""`.

Reply:
![6xnmek](https://user-images.githubusercontent.com/1288160/197021447-630a28e1-11b5-4e29-81a9-7e8bb54f7118.jpeg)